src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3286,10 +3286,12 @@ public virtual BoundNode BindMethodBody(CSharpSyntaxNode syntax, BindingDiagnost
         {
             switch (syntax)
             {
-                // PROTOTYPE(record-structs): update for record structs
                 case RecordDeclarationSyntax recordDecl:
                     return BindRecordConstructorBody(recordDecl, diagnostics);
 
+                case RecordStructDeclarationSyntax recordDecl:
+                    return BindRecordStructConstructorBody(recordDecl);
```

Comment:
>return BindRecordStructConstructorBody(recordDecl); [](start = 20, length = 51)

It doesn't feel like there is anything there ever to bind and we always can synthesize an empty body for the constructor. Perhaps we shouldn't even bother with building binders and calling them.